src/test/java/com/gachigang/ontherun/controller/DepartmentControllerTest.java

```diff
@@ -20,12 +23,46 @@ public class DepartmentControllerTest {
     private DepartmentController departmentController;
 
    @Test
-    void getDepartmentByUserId() {
+    void testGetDepartmentByUserId() {
         User user = User.builder()
                         .id(1L)
                         .lastname("test")
                         .build();
         departmentController.getDepartmentByUserId(user);
         verify(departmentService, times(1)).getDepartmentByUserId(user.getId());
     }
+
+    @Test
+    void testGetDepartmentsByBusinessIdValidId() {
```

Comment:
Honestly it depends on convenience, but foe mt it is hard to read huge camel case text, there are up to 10 naming convections, I suggest simply split main text by underscore -- for instance  testGetDepartmentsByBusinessId_ValidId -- so the first part tells us the name of testing method and the second its condition

Reply:
I've rework it